packages/mini-browser/src/browser/mini-browser-open-handler.ts

```diff
@@ -24,13 +24,14 @@ import { Command, CommandContribution, CommandRegistry } from '@theia/core/lib/c
 import { MenuContribution, MenuModelRegistry } from '@theia/core/lib/common/menu';
 import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
 import { NavigatableWidget, NavigatableWidgetOpenHandler } from '@theia/core/lib/browser/navigatable';
-import { open, OpenerService } from '@theia/core/lib/browser/opener-service';
+import { OpenerService, OpenHandler } from '@theia/core/lib/browser/opener-service';
 import { LabelProvider } from '@theia/core/lib/browser/label-provider';
 import { FrontendApplicationContribution } from '@theia/core/lib/browser/frontend-application';
 import { WidgetOpenerOptions } from '@theia/core/lib/browser/widget-open-handler';
 import { MiniBrowserService } from '../common/mini-browser-service';
 import { MiniBrowser, MiniBrowserProps } from './mini-browser';
 import { LocationMapperService } from './location-mapper-service';
+import { EditorManager } from '@theia/editor/lib/browser';
```

Comment:
You need to add the `@theia/editor` dependency in the `package.json` for the mini-browser, else the build fails.